.github/workflows/west-commands.yml

```diff
@@ -4,27 +4,25 @@ on:
   pull_request:
     branches: [main]
     paths:
+      - scripts/west_commands/sbom/**
       - scripts/west_commands/mypy.ini
       - scripts/west_commands/ncs_commands.py
       - scripts/west_commands/ncs_west_helpers.py
       - scripts/west_commands/pygit2_helpers.py
+      - scripts/requirements-west-ncs-sbom.txt
+      - scripts/requirements-extra.txt
 
 jobs:
   west_commands_job:
     runs-on: ubuntu-latest
     name: Run Python checks for west commands on patch series (PR)
     steps:
       - name: Checkout the code
-        uses: actions/checkout@v3
+        uses: nrfconnect/action-checkout-west-update@main
```

Comment:
please use fixed version

Reply:
We do not use fixed versions for actions everywhere. I prefer non-fixed at least for our internal actions. As an example we have pathed manifest-PR action multiple times without touching all the repos that use it. It is much less painful without using fixed versions...